stl/inc/xstring

```diff
@@ -2843,9 +2861,11 @@ private:
 #endif // _HAS_CXX20
                     _Traits::copy(_Unfancy(_New_ptr), _Old_ptr, _My_data._Mysize);
                     if (_My_data._Myres >= _BUF_SIZE) { // Need to deallocate old storage
+                        _ASAN_STRING_REMOVE(*this);

```

Comment:
Is that really valid? It is happening in a loop after all

Reply:
It happens in a loop, but only happens on reallocation.